python/cudf/cudf/io/text.py

```diff
@@ -14,6 +14,7 @@ def read_text(
     filepath_or_buffer,
     delimiter=None,
     byte_range=None,
+    strip_delimiters=False,
```

Comment:
Can we also update the docstring with `strip_delimiters` at https://github.com/rapidsai/cudf/blob/branch-22.12/python/cudf/cudf/utils/ioutils.py#L1205?

Reply:
done

Reply:
I don't see the commit that changes `ioutils.py`(link above), it could be a github issue..could you send me the link to your commit if you have already made the change?

Reply:
Only forgot to push after the commit :)